externs/browser/w3c_audio.js

```diff
@@ -26,33 +26,39 @@
 /**
  * @constructor
  */
-function AudioContext() {}
+function BaseAudioContext() {};
 
-/** @type {!AudioDestinationNode} */
-AudioContext.prototype.destination;
+/**
+ * @type {!AudioDestinationNode}
+ * @const
+ */
+BaseAudioContext.prototype.destination;
 
-/** @type {number} */
-AudioContext.prototype.sampleRate;
+/** @const {number} */
+BaseAudioContext.prototype.sampleRate;
 
-/** @type {number} */
-AudioContext.prototype.currentTime;
+/** @const {number} */
```

Comment:
I may have misled you a bit. I've just realized it could be a problem to mark something `@const` when its value could change, you just aren't allowed to assign to it. I've confirmed with @dimvar that although the compiler doesn't currently have any optimizations that would break code annotated that way, it might in the future.

To sum up, we don't have a good way to annotate something as `@readonly`. If its value truly won't ever change, you can use `@const`, but if it could change you just have to use `@type` to declare its type and leave it at that.

I've filed #2306 for this.